src/services/subscan.ts

```diff
@@ -0,0 +1,299 @@
+import { getSubscanBaseUrl } from '@/config/subscan-api';
+import { SubscanExtrinsicResponse } from '@/types/subscan';
+
+// 定义结果枚举
+export enum XcmMessageStatus {
+  SUCCESS = 'SUCCESS',
+  INVALID_PARA_ID = 'INVALID_PARA_ID',
+  EXTRINSIC_FAILED = 'EXTRINSIC_FAILED',
+  TIMEOUT = 'TIMEOUT',
+  UNKNOWN_ERROR = 'UNKNOWN_ERROR'
+}
+
+// 定义返回类型接口
```

Comment:
Please use English comments in the codebase, it's an open source project, available to developers around the world.